release/releaser_test.go

```diff
@@ -189,8 +233,137 @@ func TestUpdateOne(t *testing.T) {
 	if result.Status != flux.ReleaseStatusSuccess {
 		t.Errorf("expected entry to be success, but was %s", result.Status)
 	}
+}
+
+func Test_492_OnUpdateImage_OnlyShowServicePertainingToImage(t *testing.T) {
```

Comment:
We don't need the issue number in tests/comments. It'll be in the commit msg so still findable from git-blame.